### DISCUSSION TITLE
[Solved] CSS/Tailwind issue deploying on Vercel - Looking for some tips on what i'm doing wrong

### DISCUSSION THREAD

**Discussion by StewartGF** [vercel/next.js#18709] [Help] [answered]
Hi everyone, I started to play with Nextjs today, and trying to import Tailwind was a total pain, because of that, i'm looking for help.

I have this structure by default
development
--pages
---> _app.js
---> index.js
--styles
---> global.css
---> tailwilnd.css
When I run npm run dev everything looks great, but when I deploy the project to Vercel it looks kind of bad..(https://memoriae-dev-44xsohzj1.vercel.app/)

And i'm thinking it is because of global.css not being imported probably? So I don't know what's going on.
Here you have some examples of what i'm trying to say.
dev
![dev](https://i.imgur.com/hCkHycL.png)
deploy
![deploy](https://i.imgur.com/3MxyG4X.png)

If this have something to do with it, I followed this blog to import Tailwind in the project
https://medium.com/better-programming/how-to-set-up-next-js-with-tailwind-css-b93ccd2d4164

**Comment by hk86** [comment]
Have you tried (in tailwind.css)?

```
@import './global.css';

@tailwind base;
@tailwind components;
@tailwind utilities;
```

**Reply by StewartGF** [reply]
tried 
```css
/* purgecss start ignore */
@tailwind base;
@tailwind components;
@import "./global.css";
/* purgecss end ignore */
@tailwind utilities;
```

but it doesn't make any change to the page. Thanks for commenting tho, appreciate it.

**Comment by StewartGF** [comment, marked as answer]
So trying to remove piece by piece of the Tailwind setup, I found out the problem

```js
// ---------------------this
const purgecss = [
  "@fullhuman/postcss-purgecss",
  {
    content: ["./components/**/*.js", "./pages/**/*.js"],
    defaultExtractor: (content) => content.match(/[\w-/:]+(?<!:)/g) || [],
  },
];
// ---------------------this
module.exports = {
  plugins: [
    "postcss-import",
    "tailwindcss",
    "autoprefixer",
    ...(process.env.NODE_ENV === "production" ? [purgecss] : []),
  ],
};

```

removing that code made the page look the same as it was in development. So now I'm asking myself, would be any problem if I don't have that code, using just this instead:

```js
  module.exports = {
    plugins: [
      "postcss-import",
      "tailwindcss",
      "autoprefixer"
    ]
  };
```
Because from what I read with the tailwind.config.js part
```js
purge: ["./components/**/*.{js,ts,jsx,tsx}", "./pages/**/*.{js,ts,jsx,tsx}"],
```

they do the same thing
😵😵

**Comment by agcty** [comment]
This nextjs-starter has everything tailwind related correctly set up by default: https://nextjs-advanced-starter.vercel.app/
You can just clone it or use as a reference for your existing project
